lib/core/test/unit/Cardano/Wallet/Primitive/FeeSpec.hs

```diff
@@ -247,20 +253,6 @@ spec = do
             , csChngs = []
             })
 
-        -- Selection with no fee
-        feeUnitTest (FeeFixture
-            { fInps = [10,10]
-            , fOuts = [7,7]
-            , fChngs = [3,3]
-            , fUtxo = [3,3]
-            , fFee = 0
-            , fDust = 0
-            }) (Right $ FeeOutput
-            { csInps = [10,10]
-            , csOuts = [7,7]
-            , csChngs = [3,3]
-            })
```

Comment:
Now forbidden by the invariant.

Reply:
But why? Why should we disallow 0 fees?

Reply:
There are actually several issues caused by using null fees in our coin selection and fee adjustment algorithms (for instance, this could yield transactions with no inputs). Since there's no particular business requirements regarding supporting a 0-fee policy, we can save us the extra work and simply requires fees to be non-null.

Reply:
👌